sky/engine/bindings/jni/jni_array.cc

```diff
@@ -416,7 +416,7 @@ PassRefPtr<JniLongArray> JniLongArray::Create(jsize length) {
   return nullptr;
 }
 
-int64_t JniLongArray::GetArrayElement(jsize index) {
+jlong JniLongArray::GetArrayElement(jsize index) {
```

Comment:
This should continue to return int64_t (the integer type accepted by Dart's API)